index.html

```diff
@@ -4,15 +4,42 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<!--End-->
     <link rel="stylesheet" href="style.css">
     <title>Setup and Mobile first</title>
 </head>
 <body>
-    <div class="container">
-        <div class="box">B 1</div>
-        <div class="box one">B 2</div>
-        <div class="box">B 3</div>
-        <div class="box">B 4</div>
-    </div>
+    <nav class="Navbar">
+        <div class="top-logo">
+            <h2>My Portfolio</h2>
+        </div>
+        <div class="menu-icon">
+            <div class="menu-vector"></div>
+            <div class="menu-vector"></div>
+            <div class="menu-vector"></div>
+        </div>
+    </nav>
+    <main>
+        <div class="informative-top">
+            <div class="top-heading">
+                <h1>Hello, My name is Abbas Glad to see you!</h1>
+            </div>
+            <div class="heading-description">
+                <p>I'm a web developer! I can help you to create a Product or Portfolio website for you. Check out my website and contact me if you find it great.</p> 
+            </div>
+            <div class="touch-links">
+                <h3><a href="#">LET'S CONNECT</a></h3>
+                <div class="social-bar">
+                    <a href="https://twitter.com/Abbas_sDev" class="connect"><img src="image-resources/twitter.png" alt="Twitter logo"></a>
+                    <a href="https://www.linkedin.com/in/abbas-sarwar-4a0b16257/" class="connect"><img src="image-resources/linkedIn.png" alt="LinkedIn logo"></a>
+                    <a href="https://github.com/AbbasSarwar" class="connect"><img src="image-resources/Github.png" alt="Github logo"></a>
+                </div>
+            </div>
+        </div>
+    </main>
```

Comment:
- [ ] I suggest using `<section>` instead of `<main>` tag. `<section>` is a generic sectioning element, whereas `<main>` is more specific tag. Use the `<main>` tag only if the content represents the main content area of a document, otherwise use the `<section>` tag